src/graphql/main/schema.graphql

```diff
@@ -4,6 +4,7 @@ interface Account {
   displayCurrency: DisplayCurrency!
   id: ID!
   limits: AccountLimits!
+  realtimePrice: RealtimePrice!
```

Comment:
https://github.com/GaloyMoney/galoy/blob/116348a7865aec73c7a1fe4ee803302e00b006ec/src/graphql/main/schema.graphql#L977 should be marked as deprecated

Reply:
(or even removed if not used?)

Reply:
Cant be removed https://ui.honeycomb.io/galoy/datasets/galoy-bbw/result/rKUW377pxbg/a/yz7oZw5Fw5g?hideMarkers

Updated https://github.com/GaloyMoney/galoy/pull/2298/commits/3304a7639402a306e8126a52f2c8bda5b6ea5690